### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by stv0g**
Fixes https://github.com/sogno-platform/dpsim/issues/287

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/dpsim/src/Utils.h @n-eiling
+/examples @m-mirz
+/examples/villas @stv0g
+/examples/villas-deprecated @stv0g
+/gitlab-ci.yml @stv0g
+/packaging @m-mirz
+/packaging/Docker @stv0g
+/packaging/Docker/Dockerfile.dev-rocky @n-eiling
+/pyproject.toml @stv0g
+/pytest.ini @stv0g  ?
```
Comment (leonardocarreras) [inline CODEOWNERS:39]: there is an extra `?` here, maybe is still pending someone else

**Comment by m-mirz** [comment]
@stv0g When you include the comments of Ghassen, Andres and Leo it should be fine.

**Comment by stv0g** [comment]
@m-mirz Okay, thats done :) Would you approve the changes, so I can merge them?

File: CODEOWNERS (hunk 1 above)
Comment (stv0g) [reply CODEOWNERS:39]: Fixed

Comment (m-mirz) [reply CODEOWNERS:39]: @stv0g I still see the old version…

Comment (stv0g) [reply CODEOWNERS]: Oh sorry, I missed to push my changes

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator4OrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator5OrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator6aOrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator6bOrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGeneratorTrStab.cpp @gnakti
+/dpsim-models/src/SP/SP_Ph1_varResSwitch.cpp @gnakti
+/dpsim-villas/ @stv0g
+/dpsim/examples/cxx/CIM @gnakti
+/dpsim/examples/cxx/Circuits @gnakti
+/dpsim/examples/cxx/Circuits/DP_ReducedOrderSG_SMIB_Fault.cpp
```
Comment (m-mirz) [inline CODEOWNERS:57]: The owner is missing here @stv0g

Comment (stv0g) [reply CODEOWNERS]: Oups, fixed. I was blindly copying @martinmoraga proposal.

**Review by dinkelbachjan — DISMISSED** [review]

**Comment by n-eiling** [comment]
can you also add
```
/dpsim-villas/examples/cxx/FpgaExample.cpp                                     @n-eiling
```
this is a new file from a recent PR. This is why it wasn't included in the original list.

**Comment by stv0g** [comment]
> can you also add
> 
> ```
> /dpsim-villas/examples/cxx/FpgaExample.cpp                                     @n-eiling
> ```
> 
> this is a new file from a recent PR. This is why it wasn't included in the original list.

Done :)

**Review by m-mirz — APPROVED** [review]

**Pull request merged**